ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -786,17 +804,19 @@ def test018_validate_order_no_exists(self):
 
         LIMS-3406
         """
-        self.base_selenium.LOGGER.info('Running test case to check that order number should be unique')
-        order_row = self.base_selenium.get_row_cells_dict_related_to_header(self.order_page.get_random_order_row())
-
+        created_order = self.orders_api.create_new_order()
```

Comment:
why do you need to create order ?
you can use api to get random order and get it order number to use it instead of create new one

Reply:
@ghada-37 this is omar PR who is the last one working on it exactly we don't need to use the create approach so i deleted it to select random record ( Done )